web-admin/src/features/navigation/TopNavigationBar.svelte

```diff
@@ -168,7 +169,11 @@
       <ViewAsUserChip />
     {/if}
     {#if onProjectPage}
-      <ShareProjectButton {organization} {project} />
+      <ProjectAccessControls {organization} {project}>
+        <svelte:fragment slot="manage-project">
+          <UserInviteButton />
+        </svelte:fragment>
+      </ProjectAccessControls>
```

Comment:
For your awareness, @briangregoryholmes kicked off another way to manage permissions in [this PR](https://github.com/rilldata/rill/pull/5193/files). It could be more scalable than this `slot` approach – one reason being `AND`/`OR` permission logic is tricky to do with these single-permission slots.